test/net/sourceforge/kolmafia/DataFileConsistencyTest.java

```diff
@@ -621,4 +623,40 @@ public void monsterElementsAreValid() {
       fail("Couldn't read from monsters.txt");
     }
   }
+
+  @Test
+  public void everyFamiliarHasAThroneModifier() {
+    var allFamiliars =
+        FamiliarDatabase.entrySet().stream()
+            .map(Map.Entry::getKey)
+            // Ignore Pokefam-exclusive familiars
+            .filter(id -> !FamiliarDatabase.hasAttribute(id, "pokefam"))
+            // Ignore familiars with no hatchling (currently April Fools familiars, but may also
+            // catch future weirdos
+            .filter(id -> FamiliarDatabase.getFamiliarLarva(id) > 0)
+            .collect(Collectors.toSet());
+    String file = "modifiers.txt";
+    int version = 3;
```

Comment:
We should probably revive the versioned file discussion or at least consider reporting version mismatches in a more visible way..  There might be a case for bumping the version of modifiers.txt.  Neither point is blocking, however.